### ISSUE TITLE
Table definition does not update after external schema change

### ISSUE THREAD

**Issue by phpwutz** [dbeaver/dbeaver#6623]
#### System information: 
- Linux 5.2.9-arch1-1-ARCH #1 SMP PREEMPT Fri Aug 16 11:29:43 UTC 2019 x86_64 GNU/Linux
- DBeaver version 6.1.4.201909101751
- Additional extensions: none

#### Connection specification:
- MySQL [5.5.5-10.2.12-MariaDB-10.2.12+maria~xenial]
- MySQL Connector Java [mysql-connector-java-5.1.44 ( Revision: b3cda4f864902ffdde495b9df93937c3e20009be )]
- Do you use tunnels or proxies (SSH, SOCKS, etc)? - yes, creating an SSH tunnel outside of DBeaver and let DBeaver connect to localhost:$tunnelport

#### Describe the problem you're observing:

When I add a column to a table, DBeaver does not show me the new column, even after hitting refresh on every entity level (connection, database, table, row). Only restarting DBeaver eventually allows me to see the new column.

#### Steps to reproduce, if exist:
- Create a connection to the DB
- add a column to one of the tables via an interface that is NOT DBeaver
- Try to find the new column inside DBeaver
- It can not be found until DBeaver is restarted

#### Expected behavior

When changing the database schema externally (aka not via DBeaver), the "refresh" functionality should refresh the table definitions in DBeaver without restarting it.

**Comment by serge-rider** [comment]
You need to persist you table changes into the database. 
Press Save button or <kbd>CTRL+s<.kbd> after you create new column.

**Comment by phpwutz** [comment]
Hello, I am not creating the new column via DBeaver.
see my reproduction description:
- add a column to one of the tables **via an interface that is NOT DBeaver**

**Comment by serge-rider** [comment]
Yes, is misunderstood the problem.
I can't reproduce the problem you described.
Are you in transactional (manual commit) mode? Did you try to execute Commit or Rollback (main toolbar) before refreshing?

**Comment by phpwutz** [comment]
Hello, thanks again for taking the time. Apparently the conditions under which this problem occurs are more specific to my particular setup. 

I also tried creating a minimal test case and failed to reproduce. This is what I've set up (which works just fine)
- create a fresh mariadb server
- create a table with 1 column
- connect dbeaver to that database
- via CLI, add a column to the table
- hit F5 in DBeaver table/column view
- new column shows up in DBeaver (not sure wether I should put a happy or sad smiley here ^^)

I will do further investigation on my production setup (with which the problem apparently still occurs) and try to nail down the differences. If I can pinpoint the problem with an actual testcase I will open another issue.

In the meantime I will close this issue as it's clearly more specific than I thought and probably less relevant to most people.
